components/t-rex/EventfulEmbed.tsx

```diff
@@ -0,0 +1,13 @@
+import Link from "@docusaurus/Link"
+import styles from "./EventfulEmbed.module.css"
+
+const EVENTFUL_URL = new URL("https://scidev5.github.io/eventful")
+const EVENTFUL_SCHEDULE_ID = "rex"
+EVENTFUL_URL.searchParams.append("event", EVENTFUL_SCHEDULE_ID)
+
+export function EventfulEmbed() {
+    return (<div>
+        <Link className={styles.fullscreen_link} to={EVENTFUL_URL.toString()}>fullscreen <span>&#x26F6;</span></Link>
```

Comment:
If possible, I would set the `target` to a concrete value, e.g. timeline so it opens the same browser tab every time you click on it.

Reply:
added `target="_self"`